views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
```

Comment:
Hi Alex
Many people are coming to learn about Ethereum who have no technical background and have no interest themselves in building anything technical.   The primary goal of the introduction should be to welcome them and encourage them to participate in the community as writers, users, marketers, activists or whatever their talent may be.